pkg/v1/providers/infrastructure-azure/v1.0.1/cconly/overlay.yaml

```diff
@@ -0,0 +1,61 @@
+#@ load("@ytt:overlay", "overlay")
```

Comment:
@vuil Since we've merged CAPZ v1.2.1 to main, this either needs a rebase, or all Azure related changes should go in as a separate PR

Reply:
updatec